assets/src/edit-story/components/checklistCard/checklistCard.js

```diff
@@ -93,7 +93,7 @@ const ChecklistCard = ({
         </ThumbnailWrapper>
         {hasOverflowThumbnail && (
           <StyledOverflowThumbnail
-            aria-label={overflowLabel}
+            screenReaderText={overflowLabel}
```

Comment:
This is super dope
![image](https://user-images.githubusercontent.com/22185279/122787746-a07c9500-d283-11eb-885c-3fac25261cd6.png)